Add ⚡️ content to Technology Primer (#502)
BitcoinDesign/Guide: guide/getting-started/technology-primer.md

   layout = "float-right-desktop"
%}

Bitcoin, as in "I have 1 bitcoin," are just numbers in a ledger of transactions referred to as the Bitcoin blockchain. Transactions assign bitcoin from one address to another. This process reduces the available balance of the sending address and increases the balance of the receiving address. A small amount is used as the transaction fee to pay for the verification.

Therefore, owning a bitcoin means being in control of one or more addresses that have received bitcoin.

> **More info**

Reviewer: @danielnordh is using a different, sentence-based format for further reading. [For example](https://deploy-preview-501--sad-borg-390916.netlify.app/guide/designing-products/common-user-flows/#securing-a-wallet):

`See also, the Private key management section and Protecting a wallet.`

It's more compact and causes less of a visual break than the format you use with the bold "More info" and a single bullet list. How about we pick up Daniel's format? If there are 5+ links, I can imagine a bullet list work well, but for 1-2 links, the sentence works better, IMHO.

**More info**
- Node [software]({{ '/guide/getting-started/software/#nodes' | relative_url }}) and [hardware]({{ '/guide/getting-started/hardware/#nodes' | relative_url }})
</div>

## What are the benefits of this ledger?

<div class="center" markdown="1">

{% include image.html
   image = "/assets/images/guide/getting-started/technology/public.jpg"
   retina = "/assets/images/guide/getting-started/technology/public@2x.jpg"
   alt-text = "Transactions are public so can be seen and verified by anyone"
   width = 400
   height = 400
   layout = "float-right-desktop"
%}

> Imagine a global spreadsheet where all transactions get recorded, and all entries are publicly accessible. With Bitcoin, each participant can verify a transaction themself in their copy of the spreadsheet. In comparison, traditional banks keep their spreadsheets private from one another, including their customers. Besides opportunities for fraud, there are many points of failure due to this, so you can't guarantee a payment will always happen. Even though all transactions are stored publicly on Bitcoin, there is no personally identifiable information attached to them.

Reviewer: ```suggestion
Imagine a global spreadsheet where all transactions get recorded, and all entries are publicly accessible. With Bitcoin, each participant can verify a transaction for themselves in their copy of the spreadsheet. In comparison, traditional banks keep their spreadsheets private from each another and their customers. Besides opportunities for fraud, there are many points of failure due to this, so you can't guarantee a payment will always happen. Even though all transactions are stored publicly on Bitcoin, there is no personally identifiable information attached to them.
```

> With thousands of nodes and tens of millions of users worldwide, it also becomes implausible to manipulate or corrupt the previously agreed-upon transactions stored in the blockchain. To do so would require a majority of nodes to accept such change.

Reviewer: ```suggestion
With thousands of nodes and tens of millions of users worldwide, it becomes implausible to manipulate or corrupt the previously agreed-upon transactions stored in the blockchain. To do so would require a majority of nodes to accept such a change.
```

**More info**
- [Explorer software]({{ '/guide/getting-started/software/#explorers' | relative_url }})
</div>

## What is a blockchain?

<div class="center" markdown="1">

{% include image.html
   image = "/assets/images/guide/getting-started/technology/blockchain.jpg"
   retina = "/assets/images/guide/getting-started/technology/blockchain@2x.jpg"
   alt-text = "Several blocks chained together via their hashes"
   width = 400
   height = 400
   layout = "float-right-desktop"
%}

> Instead of processing each transaction individually, the Bitcoin network bundles them into [blocks]({{'/guide/glossary/#hash' | relative_url }}). A new block is created roughly every 10 minutes and can only contain a certain amount of transactions due to a strict file size limit, so transactions with the highest fees typically get chosen first. The new block must link to the one before it. Since this is a continuous process, a chain of blocks forms, hence the name blockchain.

Reviewer: ```suggestion
Instead of processing each transaction individually, the Bitcoin network bundles them into [blocks]({{'/guide/glossary/#hash' | relative_url }}). A new block is created roughly every 10 minutes. A strict file size limit results in the transactions with the highest fees typically getting chosen first. Every new block must link to the one before it. Since this is a continuous process, a chain of blocks forms, hence the name blockchain.
```

`Blocks` links to `/guide/glossary/#hash`, which does not seem right. Where should it link to?

</div>

## How is the blockchain secured?

<div class="center" markdown="1">

{% include image.html
   image = "/assets/images/guide/getting-started/technology/mining-a-block.jpg"
   retina = "/assets/images/guide/getting-started/technology/mining-a-block@2x.jpg"
   alt-text = "Transactions moving from the mempool into a block"
   width = 400
   height = 400
   layout = "float-right-desktop"
%}

While miners check that the transactions included in the block are valid, they must also be kept accountable that the information they are submitting is correct. While checking transactions for a new block, miners must also race to solve a cryptographic puzzle as proof that they have correctly done the verification. This process is called Proof of Work, and it is fundamental to the security of the blockchain. As compensation, the first miner to submit a valid block to the network, solving the puzzle, will earn all the fees from the transactions in the block and a block reward.

> We also call the Bitcoin blockchain "base layer" due to the strong security it provides.

Reviewer: A bit of a floating sentence. Not sure how strong security leads to the name "base layer".

**More info**
- Mining [software]({{ '/guide/getting-started/software/#mining-software' | relative_url }}) and [hardware]({{ '/guide/getting-started/hardware/#mining-hardware' | relative_url }})
</div>

## Do all transactions have to be this secure?

<div class="center" markdown="1">

{% include image.html
   image = "/assets/images/guide/getting-started/technology/layers.jpg"
   retina = "/assets/images/guide/getting-started/technology/layers@2x.jpg"
   alt-text = "Other networks connected to the Bitcoin blockchain"
   width = 400
   height = 400
   layout = "float-right-desktop"
%}

> Layers in Bitcoin serve to provide alternative settlement rules. Bitcoin's block generation time of 10 minutes and the 1 MB size means that the blockchain can handle around 300 000 transactions per day or 4.6 transactions per second. Layer 1, the base layer, is the most secure, so it has the most time-consuming verification process.

Reviewer: Not sure how to understand the words `Layers in Bitcoin`. Here is some alternative phrasing for this section I had noted in Figma:

> The Bitcoin network is optimized for decentralization and security and can process around 300 000 transactions per day, or 4.6 transactions per second.
> 
> This is not enough to scale as a payment method of the internet and meet the demands of our global economy. To make this possible, additional layers with different properties can be built on Bitcoin.
> 
> The Lightning network is such a second layer and allows for fast and frequent payments.

There are multiple scaling solutions built on top of Bitcoin's base layer, but we will be focusing on the Lightning Network in this guide.

**More info**
- [Transaction lifecycle](/payments/transactions)

<!--
   While the base layer needs to have high integrity of decentralization, high security guarantees, and solid monetary principals...
-->

</div>

## What is a payment channel?

<div class="center" markdown="1">

{% include image.html
   image = "/assets/images/guide/getting-started/technology/payment-channel.jpg"
   retina = "/assets/images/guide/getting-started/technology/payment-channel@2x.jpg"
   alt-text = "Several payment channels settling on the Bitcoin blockchain"
   width = 400
   height = 400
   layout = "float-right-desktop"
%}

> A payment channel is a [joint account]({{'/guide/glossary/#multi-signature-wallet-multisig' | relative_url }}) where two parties agree to bypass recording their payments on the blockchain. Opening a payment channel involves locking up some funds together in a transaction which enables faster and cheaper subsequent payments. Instead of publishing a transaction for new payments, the two parties update their balance in the payment channel by creating a transaction but don't publish it.

Reviewer: ```suggestion
A payment channel is a [joint account]({{'/guide/glossary/#multi-signature-wallet-multisig' | relative_url }}) where two parties agree to bypass recording their payments on the blockchain. Opening a payment channel involves collaboratively locking up funds in a transaction for a period of time. This enables faster and cheaper movement of those funds between the two parties as they only update their balances in the payment channel. For each update, a  new transaction is created, but it is only published when the parties agree to close the channel again.
```
Tried to edit for clarity. Do with this as you like.

</div>

> ## The Bitcoin lightning payments network

Reviewer: ```suggestion
## The Lightning payments network
```
Having the extra word Bitcoin in there seems weird. And are we capitalizing "Lightning network" or not?

<div class="center" markdown="1">

{% include image.html
   image = "/assets/images/guide/getting-started/technology/lightning-network.jpg"
   retina = "/assets/images/guide/getting-started/technology/lightning-network@2x.jpg"
   alt-text = "Lightning nodes connected to one another"
   width = 400
   height = 400
   layout = "float-right-desktop"
%}

> The lightning network is essentially a network of these payment channels. Lightning nodes allow you to have multiple channels with different parties so you route your payments though. This new network forms a second layer on top of Bitcoin.

Reviewer: ```suggestion
The Lightning network is essentially a network of these payment channels. Lightning nodes allow you to have multiple channels with different parties to route payments through. This new network forms a second layer on top of Bitcoin.
```

Lighting has some privacy benefits too since payments are not individually recorded on the blockchain. Only when the channel is opened and the final settlement transaction gets broadcasted when closing the payment channel.
</div>

## What are the ways to receive bitcoin?

<div class="center" markdown="1">

{% include image.html
   image = "/assets/images/guide/getting-started/technology/addresses.jpg"
   retina = "/assets/images/guide/getting-started/technology/addresses@2x.jpg"
   alt-text = "Examples of addresses in a bitcoin wallet"
   width = 400
   height = 400
   layout = "float-right-desktop"
%}

> To transfer bitcoin, the recipient needs to provide the sender with the destination of the payment. This is typically done in the recipient's wallet application by generating an address on the base layer or an invoice for Lightning payments. While base layer transactions are updating balances on the public ledger, Lightning invoices contain information for a payment to be routed through the network of payment channels.

Reviewer: ```suggestion
To transfer bitcoin, the recipient needs to provide the sender with the destination for the payment. On the base layer, this is typically done in the recipient's wallet application by generating and sharing an address. For Lightning payments, an invoice is used. While base layer transactions update balances on the public ledger, Lightning invoices contain information for a payment to be routed through the network of payment channels.
```
